### ISSUE TITLE
PCSA00147 Freedom wars

### ISSUE THREAD

**Issue by asawolow** [r0ah/vitacheat#3]
freedom wars cheats doesnt work...

**Comment by roahnosh** [comment]
Either version doesnt work? Try the custom dlc or the ops mod.

**Comment by roahnosh** [comment]
I'll look for the pointer code.

**Comment by andikahakim17** [comment]
hi, could you add infinite health, please? thanks before

**Comment by Smoker1** [comment]
Game Name: Freedom Wars (KO-CN)
TitleID: PCSD00087 
Version: 1.22
Available on NPS - Yes
DLC - 3
Patch - Yes (1.22 FW 3.15)

> # PCSD00087 Freedom Wars KO 1.22 MAI 4


_V0 Never Increase Sentence
$A100 8323F03C 0000BF00

_V0 Entitlement Points 9,999,999 
$A100 8323F0C4 0000BF00
$A100 8323F0E4 0000BF00

_V0 Limit Time Always Max in Battle
$A200 83001B76 BF00BF00

_V0 Infinite Sustainbility but Enermy Nope
$A100 83001BEE 0000D013
$A100 83001C02 00002000

_V0 Saved Max Civilians but Enermy Nope
$A100 8302170A 00003063
$A200 8302151A 0F00F1B9
$A200 8302151E 811AF040

_V0 Player Invincible
$A200 831A2F16 0F01F1BE 
$A100 831A2F1A 0000D011

_V0 Infinite Thorn 
$A100 8318CB08 0000BF00
$A200 8318CB34 BF00BF00

_V0 Infinite Weapon Ammo Usage
$A200 83738440 13BCF8D0 

_V0 Player High Speed Move
$3203 84701060 000000D0
$3200 00000000 00000000
$3200 00000000 00000108
$3300 00000000 40800000

_V0 Player Normal Speed Move
$3203 84701060 000000D0
$3200 00000000 00000000
$3200 00000000 00000108
$3300 00000000 3F800000

[PCSD00087.txt](https://github.com/r0ah/vitacheat/files/2427036/PCSD00087.txt)

**Comment by r0ah** [comment]
Cheat code already in the DB. Also, game is tricky. Maybe hacking the eboot is easier.

**Comment by r0ah** [comment]
Can't find any pointers. I'll try to reprogram the game with ARM code-type.

**Comment by andikahakim17** [comment]
thanks r0ah !! will wait for that 😊

**Comment by Enerccio** [comment]
PCSA00147 still NOT in database....

**Comment by RikuNoctis** [comment]
Yes, the codes take time and effort to be made, and it's not like this is a job where people get paid to do them.

You can either wait or make your own codes from scratch.

**Comment by Enerccio** [comment]
so why is the issue closed then?

**Comment by RikuNoctis** [comment]
Because it's not an issue. You are requesting something, which is completely different.

**Comment by Yohoki** [comment]
@r0ah You can't just make dumps and get pointers on this game. It's got that weird DMA that Digimon: Cyber Sleuth has, where there's 2 areas the game's data can be in. Entitlement points has no pointer, and always has an "FFFFFFFF" near it. Find the two addresses for Entitlement and always check those 2 areas to see if the "FFFFFFFF" is there or not. Kinda like this HP code:
~~~~
_V0 Inf HP
$D203 83C75A30 FFFFFFFF
$3202 839418B8 00000000
$3200 00000000 00000458
$3300 00000000 00320000
$D203 83D75A30 FFFFFFFF
$3202 83A418B8 00000000
$3200 00000000 00000458
$3300 00000000 00320000
~~~~

**Comment by AFreedomWarsPlayer** [comment]
Are there cheats available on Freedom Wars for the vita version 3.65? When I applied the cheats for vita version 3.15 the cheats will show up, yet the cheats don't work.
